### PR TITLE
get all  users

### PR DESCRIPTION
**Pull request by kerenzak**

### CODE REVIEW

File: .vscode/extensions.json
Comment (ido123ziv) [inline .vscode/extensions.json:1]: we don't need this file, remove it from git and please add to gitignore the .vscode dir

File: package-lock.json
Comment (ido123ziv) [inline package-lock.json:1]: no need for a package file for all project, can remove

File: package.json
Comment (ido123ziv) [inline package.json:1]: same

File: server/routes/api/moderator/moderatorHandler.ts [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+  }
+
+  getDBHandler() {
+    return this.dbHandler;
+  }
+
+  getAllUsers = async (req: Request, res: Response) => {
+    try {
+      const data = await this.dbHandler.getAllUsers();
+      res.status(200).json({ message: "Successfully retrieved data", data });
```
Comment (ido123ziv) [inline server/routes/api/moderator/moderatorHandler.ts:19]: if you get null from db handler you'll still return 200. let's return an error code for that (this means no users in db 😆 )

File: server/routes/api/moderator/moderatorRoutes.ts [hunk 2]
```diff
@@ -0,0 +1,11 @@
+import { Router } from "express";
+
+import Handler from "./moderatorHandler";
+
+const moderatorRouter = Router();
+
+const handler = new Handler();
+
+moderatorRouter.get("/", handler.getAllUsers);
```
Comment (ido123ziv) [inline server/routes/api/moderator/moderatorRoutes.ts:9]: thinking out load - do we want to add specific path for all users?
currently if you address: `/api/moderator/` you'll get all users maybe let's use: `/api/moderator/allUsers/`

**Review by ido123ziv — CHANGES_REQUESTED** [review]
looking great so far 🤖

File: server/routes/api/moderator/moderatorRoutes.ts (hunk 2 above)
Comment (kerenzak) [reply server/routes/api/moderator/moderatorRoutes.ts:9]: good idea

File: package-lock.json
Comment (kerenzak) [reply package-lock.json:1]: Sorry likely auto-generated files

File: package.json
Comment (kerenzak) [reply package.json:1]: sameee

**Review by ido123ziv — APPROVED** [review]

**Review by shellytesler — APPROVED** [review]
Very nice!

**Pull request merged**